docs/src/main/sphinx/release/release-458.md

```diff
@@ -0,0 +1,15 @@
+# Release 458 (11 Sep 2024)
+
+## General
+
+* Improve performance for queries with a redundant `DISTINCT` clause on an
```

Comment:
I would remove the "on an already-distinct ..." part. It's confusing and doesn't add much value to describe that.